test/functional/server/server_spec.lua

```diff
@@ -38,8 +37,8 @@ describe('serverstart(), serverstop()', function()
     eq('', eval('v:servername'))
 
     -- v:servername will take the next available server.
-    if eval('has("win32")') then
-      nvim('command', "call serverstart('\\\\.\\pipe\\test_server_pipe')")
+    if eval('has("win32")') == 1 then
```

Comment:
I would actually suggest writing this as

    local servername = (os_is_win() and [[\\.\pipe\Xtest-functional-server-server-pipe]]
                        or 'Xtest-functional-server-server-socket')
    funcs.serverstart(servername)
    eq(servername, meths.get_vvar('servername'))

. All unknown identifiers are from `helpers` package (one is missing, suggested by @justinmk).